taskcluster/fenix_taskgraph/__init__.py

```diff
@@ -13,8 +22,30 @@ def register(graph_config):
     the process.
     """
     _import_modules(["job", "worker_types", "routes", "target_tasks"])
+    extend_parameters_schema({
+        Required("head_tag"): basestring,
```

Comment:
Is it ok that this is required for branch-push and PR builds as well?

Reply:
In the FFTV branch, I tried with this being `Optional`, and I can see how there's more legwork involved. Don't mind this comment ^

Reply:
No problem! I was confused as well the first time. Thanks for trying!

Reply:
I'd be inclined to have this be `Any(text_type, None)` though.

Reply:
What's the difference with the current implementation? `head_tag` is set to an empty string, at the moment. 

I see `Any()` used at several places here https://searchfox.org/mozilla-central/rev/f1e99da78fe6c3c68696358dac06aed90f8112d3/taskcluster/taskgraph/parameters.py#75, but I don't understand the pattern.